integration/localnet/Makefile

```diff
@@ -21,6 +21,11 @@ init:
 		-consensus-delay=$(CONSENSUS_DELAY) \
 		-collection-delay=$(COLLECTION_DELAY) \
 
+# Creates a light version of the localnet with just 1 instance for each node type
+.PHONY: init-light
```

Comment:
thought this might be convenient to run instead of 
`make -e COLLECTION=1 CONSENSUS=1 EXECUTION=1 VERIFICATION=1 ACCESS=1 NCLUSTERS=1 init`
if we are running it frequently.